src/components/Wallet/index.jsx

```diff
@@ -296,47 +303,6 @@ class CreateWallet extends React.Component {
       )
   }
 
-  getUnknownAddressNodes = () => {
```

Comment:
These two methods were moved into selectors. This should save on work being done on each component re-render.